agasc/tests/test_multi_agasc_file.py

```diff
@@ -0,0 +1,40 @@
+import os
+import tables
+from ska_path import ska_path
+from astropy.table import Table
+import tempfile
+
+from .. import agasc
+
+DATA_ROOT = ska_path('data', 'agasc')
+tables_open_file = getattr(tables, 'open_file', None) or tables.openFile
+
+def test_multi_agasc():
+
+    tempdir = tempfile.mkdtemp()
+
+    # Make two custom agasc files from the miniagasc, using 20 stars from
+    # around the middle of the table
+    with tables_open_file(os.path.join(DATA_ROOT, 'miniagasc.h5')) as h5:
+        middle = len(h5.root.data) / 2
+        stars1 = Table(h5.root.data[middle:middle+20])
+        stars1.write(os.path.join(tempdir, 'stars1.h5'), path='data')
```

Comment:
h5py should be added to ska3.  I'm don't recall why it isn't already there since dumping a table like this is quite useful.